.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+# Copyright 2024 The Authors (see AUTHORS file)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'release'
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/**/*'
+
+jobs:
+  release:
+    runs-on: 'ubuntu-latest'
+    if: '${{ startsWith(github.event.head_commit.message, ''Release: v'') && github.event.head_commit.author.email == vars.TOKEN_MINTER_GITHUB_EMAIL }}'
```

I prefer the following syntax for readability:

```suggestion
    if: |-
      startsWith(github.event.head_commit.message, 'Release: v') &&
      github.event.head_commit.author.email == vars.TOKEN_MINTER_GITHUB_EMAIL
```